quality-metrics/qm-pull-requests.md

```diff
@@ -0,0 +1,25 @@
+---
+Author: John Bellone <jbellone@bloomberg>
+Status: Draft
+License: Apache 2.0
+---
+
+## Cookbook has closed pull-requests on GitHub.
```

Comment:
this is a good reason to think about metrics that are not applicable.

`source_url` will tell us where the development repo is.  With that, we can then build integrations into various systems.  It makes sense to start with GitHub.

If `source_url` points to a public GitHub repo we should be able to look at PRs, issues, date of last commit, etc.

As we iterate additional remotes could be added to the system.